packages/ember-templates/lib/index.js

```diff
@@ -21,6 +29,39 @@ Ember.TextField = TextField;
 Ember.TextArea = TextArea;
 Ember.LinkComponent = LinkComponent;
 
+if (Ember.HTMLBars) {
+  Ember.HTMLBars.SafeString = SafeString;
+} else {
+  Ember.HTMLBars = { SafeString };
+}
+
+if (Ember.HandlerBars) {
+  Ember.HandlerBars.SafeString = SafeString;
+  if (Ember.HandlerBars.Utils) {
+    Ember.HandlerBars.Utils.escapeExpression = escapeExpression;
+  } else {
+    Ember.HandlerBars.Utils = { escapeExpression };
+  }
+} else {
+  Ember.HandlerBars = { SafeString, Utils: { escapeExpression } };
+}
+
+if (Ember.String) {
+  Ember.String.Utils = { htmlSafe };
```

Comment:
This should be `Ember.String.htmlSafe` (no `Utils`).  Can you also import `String` from `ember-runtime`?